src/index.html

```diff
@@ -12,6 +12,142 @@
   </head>
   <body>
     <h1>HTML Form</h1>
+    <form
+      onsubmit="onSubmit()"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            id="surname"
+            name="surname"
+            required>
```

Comment:
Same here. Fix it everywhere in your project.
```suggestion
          <input
            type="text"
            id="surname"
            name="surname"
            required
          >
```